wire/cache_internal_test.go

```diff
@@ -15,14 +15,25 @@
 package wire
 
 import (
+	"math/rand"
 	"testing"
 
 	"github.com/stretchr/testify/assert"
 	"github.com/stretchr/testify/require"
 
+	wtest "perun.network/go-perun/wallet/test"
 	"polycry.pt/poly-go/test"
 )
 
+// NewRandomEnvelope - copy from wire/test for internal tests.
```

Comment:
```suggestion
// NewRandomEnvelope - copy from wire/test for internal tests.
```

This comment looks like a leftover. Either just explain what the function does or remove.

Reply:
The comment isn't a left over. The function is originally defined in `wire/test`. Because, `wire` cannot import `wire/test`, it is duplicated here.

Reply:
The point is that you are making a reference to an external package. I don't think it's very useful. You could try to place that function somewhere where both locations can use it. Anyways, up to you.

Reply:
The only place to place it where both packages could use would be `wire` package. But, that will pollute public API of `wire` package. So, I think this is an optimal middle ground.